src/components/Onboarding/OnboardingScreen.js

```diff
@@ -0,0 +1,668 @@
+          fullWidth
+          name="confirmPassword"
+          id="standard-password-input"
+          value={formState.confirmPassword}
+          label="Confirm Password"
+          onChange={handleChange}
+        />
+      </Grid>
+      {errorMsg && <Grid item xs={12} className="error-msg">{errorMsg}</Grid>}
+      {repeatPwdCheck && <Grid item xs={12} className="error-msg">{repeatPwdCheck}</Grid>}
+      <Grid item xs={12} sm={6}>
+        <Button
+          type="button"
+          onClick={onPrev}
+          fullWidth
+          color="primary"
+          variant="contained"
+          className={classes.blackButton}
+        >
+          Back
+        </Button>
+      </Grid>
+      <Grid item xs={12} sm={6}>
+        <Button
+          className={classes.greenButton}
+          type="button"
+          onClick={onNext}
+          fullWidth
+          color="primary"
+          variant="contained"
+          disabled={isInvalid}
+        >
+          Next
+        </Button>
+      </Grid>
+    </Grid>,
+  ];
+}
+
+function Step3({
+  currentStep, formState, classes, role, handleChange, onPrev, handleSubmit, errorMsg,
+  step3Check, step3IsInvalid, loading,
+}) {
+  const COMPANY_NAMES = ['Company Name', 'Agency', 'Farm Name'];
+  if (currentStep !== 3) {
+    return null;
+  }
+  return [
+    <div className={classes.subtitleText}>
+      Contact Information
+    </div>,
+    <Grid
+      container
+      spacing={2}
+      alignItems="center"
+    >
+      <Grid item xs={12}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          name="org"
+          label={COMPANY_NAMES[role]}
+          value={formState.org}
+          onChange={handleChange}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          label="Location"
+          name="zipcode"
+          value={formState.zipcode}
+          onChange={handleChange}
+        />
+      </Grid>
+      <Grid item xs={12} sm={6}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          id="fname"
+          label="First Name"
+          name="firstName"
+          placeholder="First Name"
+          value={formState.firstName}
+          onChange={handleChange}
+        />
+      </Grid>
+      <Grid item xs={12} sm={6}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          id="lname"
+          name="lastName"
+          placeholder="Last Name"
+          value={formState.lastName}
+          onChange={handleChange}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          label="Phone"
+          name="phone"
+          value={formState.phone}
+          onChange={handleChange}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
```

Comment:
Is there a way to remove the asterisk next to comments in the text input client side? I'm looking at the web app rn